index.js

```diff
@@ -138,47 +188,64 @@ function nowServing(line) {
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
+let tickCounter = 1
 
+function takeATicketNumber(line) {
+  line.push(tickCounter)
+  tickCounter ++
+  return `Welcome. You are ticket number ${line[line.length-1]}.`
+}
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
+console.log(takeATicketNumber(line))
+// => `Welcome. You are ticket number 1`
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 2`
+console.log(takeATicketNumber(line))
+// => `Welcome. You are ticket number 2`
 
-// console.log(nowServing(line))
-// // => `Currently serving 1.`
+console.log(nowServing(line))
+// => `Currently serving 1.`
 
-// console.log(nowServing(line))
-// // => `Currently serving 2.`
+console.log(nowServing(line))
+// => `Currently serving 2.`
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 3`
+console.log(takeATicketNumber(line))
+// => `Welcome. You are ticket number 3`
 
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 // ***** Scope & Closures - Question 2 *****
+
+function ticketNumberGeneratorFunc () {
+  let counterNew = 1
+  function takeATicketNumberNew() {
+    newLine.push(counterNew)
+    counterNew ++
+    return `Welcome. You are ticket number ${newLine[newLine.length-1]}.`
+  }
+  return takeATicketNumberNew;
+}
```

Comment:
Good work! Line 226 can be simplified similar to the comment I left on the previous question.